themes/ssp/core/no_cookie.tpl.php

```diff
@@ -10,13 +10,13 @@
 $this->data['header'] = $header;
 $this->includeAtTemplateBase('includes/header.php');
 
-echo('<h2>' . $header . '</h2>');
-echo('<p>' . $description . '</p>');
+echo ('<h2>' . $header . '</h2>');
+echo ('<p>' . $description . '</p>');
 
-if ($retryURL !== NULL) {
-	echo('<ul class="list-unstyled">');
-	echo('<li><a href="' . htmlspecialchars($retryURL) . '" id="retry">' . $retry . '</a></li>');
-	echo('</ul>');
+if ($retryURL !== null) {
```

Comment:
Should we remove all echo and replace with
`<?= $header ?>` like syntax

Reply:
Resolved by commit [Remove sequentially echos ](https://github.com/EGI-Federation/simplesamlphp-module-themeegi/pull/14/commits/0d8e00ed36955b6ffb716baef99a40a281115645)